### ISSUE TITLE
snapshots/20210630

### ISSUE THREAD

**Issue by dvdhrm** [osbuild/rpmrepo#10]
This is a tracking issue for the scheduled snapshots targetting 20210630.

**Comment by dvdhrm** [comment]
`Job "daherrma-snapshot-20210630" with id "ccab4469-cc77-43ee-b06c-a2b93fd92679" will be submitted`

All 71 jobs are running.

**Comment by dvdhrm** [comment]
All 71 jobs succeeded.

**Comment by dvdhrm** [comment]
```sh
curl https://rpmrepo.osbuild.org/v2/enumerate | jq '[.[] | select(test("20210630"))]'
[
  "cs8-aarch64-appstream-20210630",
  "cs8-aarch64-baseos-20210630",
  "cs8-ppc64le-appstream-20210630",
  "cs8-ppc64le-baseos-20210630",
  "cs8-x86_64-appstream-20210630",
  "cs8-x86_64-baseos-20210630",
  "el8-aarch64-appstream-n8.4-20210630",
  "el8-aarch64-appstream-n8.5-20210630",
  "el8-aarch64-appstream-r8.2-20210630",
  "el8-aarch64-appstream-r8.3-20210630",
  "el8-aarch64-baseos-n8.4-20210630",
  "el8-aarch64-baseos-n8.5-20210630",
  "el8-aarch64-baseos-r8.2-20210630",
  "el8-aarch64-baseos-r8.3-20210630",
  "el8-ppc64le-appstream-n8.4-20210630",
  "el8-ppc64le-appstream-n8.5-20210630",
  "el8-ppc64le-appstream-r8.2-20210630",
  "el8-ppc64le-appstream-r8.3-20210630",
  "el8-ppc64le-baseos-n8.4-20210630",
  "el8-ppc64le-baseos-n8.5-20210630",
  "el8-ppc64le-baseos-r8.2-20210630",
  "el8-ppc64le-baseos-r8.3-20210630",
  "el8-s390x-appstream-n8.4-20210630",
  "el8-s390x-appstream-n8.5-20210630",
  "el8-s390x-appstream-r8.2-20210630",
  "el8-s390x-appstream-r8.3-20210630",
  "el8-s390x-baseos-n8.4-20210630",
  "el8-s390x-baseos-n8.5-20210630",
  "el8-s390x-baseos-r8.2-20210630",
  "el8-s390x-baseos-r8.3-20210630",
  "el8-x86_64-appstream-n8.4-20210630",
  "el8-x86_64-appstream-n8.5-20210630",
  "el8-x86_64-appstream-r8.2-20210630",
  "el8-x86_64-appstream-r8.3-20210630",
  "el8-x86_64-baseos-n8.4-20210630",
  "el8-x86_64-baseos-n8.5-20210630",
  "el8-x86_64-baseos-r8.2-20210630",
  "el8-x86_64-baseos-r8.3-20210630",
  "el8-x86_64-google-cloud-sdk-20210630",
  "el8-x86_64-google-compute-engine-20210630",
  "el8-x86_64-rt-n8.4-20210630",
  "el8-x86_64-rt-n8.5-20210630",
  "el8-x86_64-rt-r8.2-20210630",
  "el8-x86_64-rt-r8.3-20210630",
  "el9-aarch64-appstream-n9.0-20210630",
  "el9-aarch64-baseos-n9.0-20210630",
  "el9-ppc64le-appstream-n9.0-20210630",
  "el9-ppc64le-baseos-n9.0-20210630",
  "el9-s390x-appstream-n9.0-20210630",
  "el9-s390x-baseos-n9.0-20210630",
  "el9-x86_64-appstream-n9.0-20210630",
  "el9-x86_64-baseos-n9.0-20210630",
  "el9-x86_64-rt-n9.0-20210630",
  "f33-aarch64-updates-released-20210630",
  "f33-aarch64-updates-released-modular-20210630",
  "f33-x86_64-updates-released-20210630",
  "f33-x86_64-updates-released-modular-20210630",
  "f34-aarch64-updates-released-20210630",
  "f34-aarch64-updates-released-modular-20210630",
  "f34-x86_64-updates-released-20210630",
  "f34-x86_64-updates-released-modular-20210630",
  "f35-x86_64-rawhide-20210630",
  "f35-x86_64-rawhide-modular-20210630"
]
```
